src/earn/saga.ts

```diff
@@ -386,7 +390,10 @@ export function* withdrawSubmitSaga(action: PayloadAction<WithdrawInfo>) {
           providerId: pool.appId,
         }
       }
-      createWithdrawStandbyTxHandlers.push(createClaimRewardStandbyTx)
+
+      if (!pool.dataProps.withdrawalIncludesClaim) {
```

Comment:
This is a small fix to address an issue of additional claim transactions being included when withdrawals include a claim.

Reply:
please add regression tests when you fix bugs like this

Reply:
Added a test in 7c5d529d1b46d70f8b8cf32f073a1a6e8866377b!